instrumentation/okhttp/okhttp-3.0/agent/build.gradle.kts

```diff
@@ -11,6 +11,6 @@ android {
 
 dependencies {
     implementation(project(":instrumentation:okhttp:okhttp-3.0:library"))
-    implementation(libs.okhttp)
+    implementation(libs.okhttp) // TODO - should this be compileOnly?
```

Comment:
@breedx-splk what do you think about this?

Reply:
I think I agree that it should be compileOnly. :)